test_scripts/Polices/build_options/ATF_P_Register_NewApp_not_exist_inLocalPT_FinishPTU_PROPRIETARY.lua

```diff
@@ -92,6 +98,14 @@ function Test:TestStep_RegisterNewApplication()
   EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = "Media Application" }})
   self.mobileSession2:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
   self.mobileSession2:ExpectNotification("OnPermissionsChange")
+  self.mobileSession2:ExpectNotification("OnSystemRequest")--, {requestType = "LOCK_SCREEN_ICON_URL"} )
+  :ValidIf(function(_,data)
+      if(data.payload.requestType ~= "LOCK_SCREEN_ICON_URL") then
+        commonFunctions:printError("requestType should be PROPRIETARY")
```

Comment:
@istoimenova, Test step will fail in case _requestType_ is not equal to "LOCK_SCREEN_ICON_URL"
Ex.: case will fail if "PROPRIETARY" is received. So why we print "requestType should be PROPRIETARY" though?